client/components/Planning/PlanningEditor/index.jsx

```diff
@@ -568,6 +603,30 @@ export class PlanningEditorComponent extends React.Component {
                             onFocus={onFocusDetails}
                         />}
                     </ToggleBox>
+
+                    <ToggleBox
```

Comment:
Please use the `planning_types` collection to control if `Files` are to be available for `Planning Items`

Reply:
fixed